draft-ietf-avtcore-rtp-over-quic.md

```diff
@@ -742,6 +742,14 @@ RTP retransmissions can be done in the same RTP session or in a
 different RTP session {{!RFC4588}} and the flow identifier MUST be set to the
 flow identifier of the RTP session in which the retransmission happens.
 
+## RTCP
+
+The same encapsulation as described above for RTP packets can also be used to carry RTCP packets back from the receiver to the sender. Similar to the RTP packet RTCP can use datagrams or streams as well.
+If a receiver sends aggregated RTCP reports for multiple RTP streams the flow identifier no longer machtes the flow identifier for a single RTP stream. Thus the sender always needs to inspect the received RTCP packet independent of the flow identifier used to the RTCP flow to determine to which of the RTP flows the received packets apply.
+This is also the reason why bidirectional streams are not recommendend, as the the received RTCP packets would not necessarily apply to the same RTP stream being send on the same flow.
+Further more using a bidirectional stream could result in a situation where the sender has closed its side of the QUIC stream, but the receiver continues to send RTCP in the opposite direction.
+Thus it makes more sense if the sender and receiver agree on one or multiple unidirectional streams to transport any RTCP messages.
+
```

Comment:
I'm not sure whether we need the text about bidirectional streams, using them would be a protocol error.  That's from [Section 5.2](https://mengelbart.github.io/rtp-over-quic-draft/draft-ietf-avtcore-rtp-over-quic.html#name-quic-streams) - bidirectional streams are a ROQ_STREAM_CREATION_ERROR.

I think we still need the final sentence in the new text. @mengelbart, what do you think?

```suggestion
If a receiver sends aggregated RTCP reports for multiple RTP streams, the flow identifier no longer matches the flow identifier for a single RTP stream. Thus the sender always needs to inspect the received RTCP packet independent of the flow identifier used to the RTCP flow to determine to which of the RTP flows the received packets apply.
This is also the reason why bidirectional streams are not allowed, as the received RTCP packets would not necessarily apply to the same RTP stream being sent on the same flow.
In addition, allowing a bidirectional stream could result in a situation where the sender has closed its side of the QUIC stream, but the receiver continues to send RTCP in the opposite direction.
Thus it makes more sense if the sender and receiver agree on one or multiple unidirectional streams to transport any RTCP messages.

```

Reply:
I agree that it is a protocol violation to use bidirectional streams, but I think it is helpful to have this explanation as to why it doesn't make sense to use bidirectional streams to relate RTCP packets to RTP packets. Maybe change the words `not recommendend` to something else because it is not only not recommend, but forbidden to use bidirectional streams?

Reply:
@mengelbart Spencer changed the "not recommended" already into "not allowed". Are we happy with that, or should I try to come up with a better phrasing?

Reply:
Sorry for the slow response. I'm happy with applying Spencer's suggestion and then merging this.